azure-functions/acbs-function/api.js

```diff
@@ -158,30 +191,36 @@ const patch = async (endpoint, payload, eTag) => {
  * HTTP methods and endpoints.
  */
 
+// *** MDM ***
+
+// POST
+const createFacilityCovenantId = (payload) => postMdm('numbers', payload);
+
+// *** TFS ***
+
 // GET
-const getFacility = (facilityId) => get(`facility/${facilityId}`);
-const getLoanId = (facilityId) => get(`facility/${facilityId}/loan`);
+const getFacility = (facilityId) => get(`facilities/${facilityId}`);
+const getLoanId = (facilityId) => get(`facilities/${facilityId}/loan`);
```

Comment:
Small typo here, it should be `/loans` at the end of the URL :)

Reply:
Have added `s`.